libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md

```diff
@@ -25,8 +26,8 @@ The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
 receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
 
  - SERIAL0 -> USB
- - SERIAL1 -> UART1 (DMA-enabled)
- - SERIAL2 -> UART2 (RCIN one wire, DMA-enabled)
+ - SERIAL1 -> UART1 (DMA-enabled) RCin
+ - SERIAL2 -> UART2 (SmartPort, DMA-enabled)
```

Comment:
only onewire here, probably should make that clear

Reply:
added